exir/memory_planning.py

```diff
@@ -339,6 +332,22 @@ def _is_mutable_buffer(
     return False
 
 
+def _does_user_inputs_exist(graph_signature: Optional[ExportGraphSignature]) -> bool:
```

Comment:
nit: subject-verb agreement, should be either `_does_user_input_exist` or `_do_user_inputs_exist`

Reply:
touché 😅